client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -270,6 +270,59 @@ function MetadataListEditingDirective() {
     };
 }
 
+MetadataSliderDirective.$inject = ['desks'];
+function MetadataSliderDirective(desks) {
+    return {
+        scope: {
+            list: '=',
+            disabled: '=ngDisabled',
```

Comment:
Yes, I use this component in header-info.html, and I plan to make it global, so we can use it anywhere.